src/transformers/file_utils.py

```diff
@@ -1022,42 +1024,57 @@ def _prepare_output_docstrings(output_type, config_class, min_indent=None):
 
     >>> outputs = model(**inputs, start_positions=start_positions, end_positions=end_positions)
     >>> loss = outputs.loss
+    >>> round(loss.item(), 2)
+    {expected_loss}
```

Comment:
I went over this PR too quickly :-/ Why didn't we check for the actual answer of the model here? 

The model is a QA model -> it's not very descriptive to just test for the output shape